pkg/mountkexec/mountkexec.go

```diff
@@ -52,13 +52,19 @@ func MountISOPmem(pmem, mountDir string) error {
 func KexecISO(opp *webboot.Distro, dir string) error {
 	var image boot.OSImage
 	kernelPath := opp.Kernel
+
+	kernel, err := os.Open(kernelPath)
+
+	if err != nil {
+		fmt.Errorf("Error: %v", err)
```

Comment:
Get rid of the Error: OR make it a lot more meaningful:
KexecISO:

Reply:
So i guess no error necessary, because multiboot.Probe won’t work if kernel doesn’t.